nsls2tools/csx1/startup/settings.py

```diff
@@ -0,0 +1,95 @@
+from bluesky.magics import BlueskyMagics
+
+from .startup import sd
+from .detectors import *
+from .endstation import *
+from .accelerator import *
+from .optics import *
+from .tardis import *
+
+#
+# Setup of sup. data for plans
+#
+sd.monitors = []
+sd.flyers = []
+sd.baseline = [theta, delta, gamma,
+               sx, say, saz,
+               cryoangle, sy, sz,
+               epu1, epu2,
+               slt1, slt2, slt3,
+               m1a, m3a,
+               nanop, tardis,
+               stemp, pgm]
+
+sclr.names.read_attrs=['name1','name2','name3','name4','name5','name6']  # TODO  WHAT IS THIS??? - Dan Allan
+sclr.channels.read_attrs=['chan1','chan2','chan3','chan4','chan5','chan6']
+sclr.hints = {'fields': ['sclr_ch2', 'sclr_ch3', 'sclr_ch6']}
+
+def relabel_fig(fig, new_label):
+    fig.set_label(new_label)
+    fig.canvas.manager.set_window_title(fig.get_label())
+
+fccd.hints = {'fields': ['fccd_stats1_total']}
+dif_beam.hints = {'fields' : ['dif_beam_stats3_total','dif_beam_stats1_total']}
+
+# This was imported in 00-startup.py #  used to generate the list: [thing.name for thing in get_all_positioners()]
+
+BlueskyMagics.positioners = [
+    cryoangle,
+    delta,
+    diag2_y,
+    diag3_y,
+    diag5_y,
+    diag6_pid,
+    diag6_y,
+    epu1.gap,
+    epu1.phase,
+    epu2.gap,
+    epu2.phase,
+    es_diag1_y,
+    eta,
+    gamma,
+    m1a.z,
+    m1a.y,
+    m1a.x,
+    m1a.pit,
+    m1a.yaw,
+    m1a.rol,
+    m3a.x,
+    m3a.pit,
+    m3a.bdr,
+    # muR,  # TODO turn this back on when safe
```

Comment:
Is this safe to turn back on? I see that `muR` and `muT` are defined in `tardis.py` now so my guess is that uncommenting this would be safe to do.

Reply:
It should be.  Right now the motors are not plugged in so no one can move it until I am ready to test it.  Which would mean that we would need to enable it.

@stuwilkins if I were to want to update this, should I:
* work on your fork once back at NSLS-II
* take my fork of nsls-ii-tool and fetch your fork onto mine?
* make a fork of your fork (which seems a bit complicated) but maybe this is a good way to force a LBS review of the request before the big merge back into the facility